Source/ExecutionEngine/VerificationTask.cs

```diff
@@ -7,6 +7,7 @@
 using System.Reactive.Subjects;
 using System.Runtime.CompilerServices;
 using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Do you use that?

Reply:
Not sure. I'm not too concerned about unused usings though.